core/domain/topic_services.py

```diff
@@ -798,7 +799,7 @@ def compute_summary_of_topic(topic):
 
     topic_summary = topic_domain.TopicSummary(
         topic.id, topic.name, topic.canonical_name, topic.language_code,
-        topic.version, topic_model_canonical_story_count,
+        topic.description, topic.version, topic_model_canonical_story_count,
```

Comment:
Shouldn't there be an edit to topic_services_test as well, to check this?

Reply:
Yeah there should be. Updated the tests.